ui/imports/shared/views/TabAddressSelectorView.qml

```diff
@@ -77,46 +90,34 @@ Item {
                 height: Math.min(d.maxHeightForList, savedAddresses.contentHeight)
 
                 model: root.store.savedAddressesModel
-                header:  savedAddresses.count > 0 ? search : nothingInList
+                header: savedAddresses.count > 0 ? search : nothingInList
                 headerPositioning: ListView.OverlayHeader
-                delegate: StatusListItem {
-                    implicitWidth: parent.width
-                    height: visible ? 64 : 0
-                    title: name
-                    subTitle: address
-                    radius: 0
-                    color: sensor.containsMouse || highlighted ? Theme.palette.baseColor3 : "transparent"
+                delegate: SavedAddressListItem {
+                    implicitWidth: ListView.view.width
+                    modelData: model
                     visible: !savedAddresses.headerItem.text || name.toLowerCase().includes(savedAddresses.headerItem.text)
-//                    TODO uncomment when #6456 is fixed
-//                    components: [
-//                        StatusIcon {
-//                            icon: "star-icon"
-//                            width: 12
-//                            height: 12
-//                        }
-//                    ]
-                    onClicked: contactSelected(address, RecipientSelector.Type.Address )
+                    onClicked: recipientSelected(modelData, TabAddressSelectorView.Type.SavedAddress)
                 }
                 Component {
                     id: search
-                    ColumnLayout {
+                    StatusInput {
```

Comment:
`SearchBox`?

Reply:
Sadly what I need is 
![image](https://user-images.githubusercontent.com/60327365/233945687-2d23613d-42b9-4372-b054-73fe107a1bbc.png)
as per design.
The SearchBox looks like this
<img width="529" alt="image" src="https://user-images.githubusercontent.com/60327365/233945915-5e7eddb3-4c67-4110-9fd4-a83e031f4f70.png">
so they arent matching, however I use them in a few places in the WalletSection and maybe I can create a new component extending the SearchBox, which handles what I need.

Reply:
Would be cool, thanks!